app/packs/src/apps/admin/DelayedJobs.js

```diff
@@ -52,14 +51,14 @@ export default class DelayedJobs extends Component {
 
     const tcolumn = (
       <tr className="align-middle">
-        <th className="w-4 fs-4" colSpan="2">ID</th>
-        <th className="w-5 fs-4">Queue</th>
-        <th className="w-5 fs-4">Job Class</th>
-        <th className="w-5 fs-4">Run At</th>
-        <th className="w-5 fs-4">Failed At</th>
-        <th className="w-4 fs-4">Attempts</th>
-        <th className="w-4 fs-4">Priority</th>
-        <th className="w-50 fs-4">Last Errors</th>
+        <th className="w-4" colSpan="2">ID</th>
+        <th className="w-5">Queue</th>
+        <th className="w-5">Job Class</th>
+        <th className="w-5">Run At</th>
+        <th className="w-5">Failed At</th>
+        <th className="w-4">Attempts</th>
+        <th className="w-4">Priority</th>
+        <th className="w-50">Last Errors</th>
       </tr>
```

Comment:
The same here. w-50 exists
Just take a look how the table looks like without w-4 and w-5